Remove mostly empty tutorials and codelabs section from sidenav (#5858)
dart-lang/site-www: src/content/tutorials/index.md

These tutorials teach you how to use the Dart
language, tools, and APIs to build applications.

> ## Learn the basics {:#basics}

Reviewer: **issue:** Fix lines 14 and 15 below.

```markdown
To learn about futures, try the asynchronous programming tutorial.
```

</div>

<a id="server-side-dart-tutorials" aria-hidden="true"></a>
## Build server-side apps {:#server}

{% include 'server-tutorials.md' %}

<a id="more-tutorials" aria-hidden="true"></a>
## Keep learning

> Follow more tutorials and codelabs to get more hands-on experience

Reviewer: **issue:** Same as [line 2778](https://github.com/dart-lang/site-www/pull/5858/files#diff-75ed5ce4cf7ea66ddd2f5aa89546afe28498b1afb33bcf6a6f56fb7407445cd6R2778) in [js-to-dart.md](src/content/resources/coming-from/js-to-dart.md)

```suggestion
To get more hands-on experience, try more of the available tutorials
```
